packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -508,6 +510,34 @@ function AuthOptions({
             )}
           </EmailVerified>
         </Tab>
+        <Tab label={AuthDisplay.OnboardingSignupV4d5}>
+          <OnboardingRegistrationForm4d5
```

Comment:
Was there any way we can adjust the `default` to become this?
(can be done later as you suggested)